lib/mix/lib/mix/exceptions.ex

```diff
@@ -61,7 +61,7 @@ defmodule Mix.ElixirVersionError do
 end
 
 defmodule Mix.NoProjectError do
-  message = "Could not find a Mix.Project, please ensure Mix is running on a directory with a mix.exs"
+  message = "Could not find a Mix.Project, please ensure Mix is running in a directory with a mix.exs"
```

Comment:
On second look, I'm wondering if this should actually say:
"Could not find a Mix.Project, please ensure you are running Mix in a directory with a mix.exs file"
?